dll/shellext/netshell/lanstatusui.cpp

```diff
@@ -819,7 +819,21 @@ ShowStatusPropertyDialog(
         else if (pProperties->Status == NCS_MEDIA_DISCONNECTED || pProperties->Status == NCS_DISCONNECTED ||
                  pProperties->Status == NCS_HARDWARE_DISABLED)
         {
-            ShowNetConnectionProperties(pContext->pNet, pContext->hwndDlg);
+            WCHAR buffer[100];
+            
+            LoadStringW(netshell_hInstance, IDS_NETWORKCONNECTION, buffer, _countof(buffer));
+            HWND prophwnd = FindWindow(NULL, buffer);
```

Comment:
This makes assumptions about how Explorer formats its window titles? `IShellWindows` is a lot more work so I guess we can let it slide for now.